code/WorkInProgress/recycling/conveyor.dm

```diff
@@ -764,6 +970,38 @@ ADMIN_INTERACT_PROCS(/obj/machinery/conveyor_switch, proc/trigger)
 				C.setdir()
 		SEND_SIGNAL(src,COMSIG_MECHCOMP_TRANSMIT_SIGNAL,"switchTriggered")
 
+/obj/machinery/conveyor_switch/built/
```

Comment:
Why is this a separate type? Also the clicking on one and then the other mechcomp behaviour still works and overrides the panel openness.

Reply:
So it can spawn with it's cover open. Will try to fix that as soon as I get home.